src/mappings/discord/utils/ComponentDispatchUtils.ts

```diff
@@ -0,0 +1,24 @@
+import { ModuleExportType } from "@moonlight-mod/moonmap";
+import register from "../../../registry";
+
```

Comment:
I realized our documentation for writing types was outdated, so I fixed that: https://moonlight-mod.github.io/dev/mappings/#contributing-upstream

You should probably do this if you're going to import it.